docs/source/statistics.mdx

```diff
@@ -0,0 +1,405 @@
+        {
+            headers: { Authorization: `Bearer ${API_TOKEN}` },
+            method: "GET"
+        }
+    );
+    const result = await response.json();
+    return result;
+}
+query().then((response) => {
+    console.log(JSON.stringify(response));
+});
+```
+</js>
+<curl>
+```curl
+curl https://datasets-server.huggingface.co/statistics?dataset=glue&config=cola&split=train \
+        -X GET \
+        -H "Authorization: Bearer ${API_TOKEN}"
+```
+</curl>
+</inferencesnippet>
+
+The response JSON contains two keys:
+* `num_examples` - number of samples in a split
+* `statistics` - list of dictionaries of statistics per each column, each dictionary has three keys: `column_name`, `column_type`, and `column_statistics`. Content of `column_statistics` depends on a column type, see [Response structure by data types](./statistics#response-structure-by-data-type) for more details
+
+```json
+{
+  "num_examples": 8551,
+  "statistics": [
+    {
+      "column_name": "idx",
+      "column_type": "int",
+      "column_statistics": {
+        "nan_count": 0,
+        "nan_proportion": 0,
+        "min": 0,
+        "max": 8550,
+        "mean": 4275,
+        "median": 4275,
+        "std": 2468.60541,
+        "histogram": {
+          "hist": [
+            856,
+            856,
+            856,
+            856,
+            856,
+            856,
+            856,
+            856,
+            856,
+            847
+          ],
+          "bin_edges": [
+            0,
+            856,
+            1712,
+            2568,
+            3424,
+            4280,
+            5136,
+            5992,
+            6848,
+            7704,
+            8550
+          ]
+        }
+      }
+    },
+    {
+      "column_name": "label",
+      "column_type": "class_label",
+      "column_statistics": {
+        "nan_count": 0,
+        "nan_proportion": 0,
+        "no_label_count": 0,
+        "no_label_proportion": 0,
+        "n_unique": 2,
+        "frequencies": {
+          "unacceptable": 2528,
+          "acceptable": 6023
+        }
+      }
+    },
+    {
+      "column_name": "sentence",
+      "column_type": "string_text",
+      "column_statistics": {
+        "nan_count": 0,
+        "nan_proportion": 0,
+        "min": 6,
+        "max": 231,
+        "mean": 40.70074,
+        "median": 37,
+        "std": 19.14431,
+        "histogram": {
+          "hist": [
+            2260,
+            4512,
+            1262,
+            380,
+            102,
+            26,
+            6,
+            1,
+            1,
+            1
+          ],
+          "bin_edges": [
+            6,
+            29,
+            52,
+            75,
+            98,
+            121,
+            144,
+            167,
+            190,
+            213,
+            231
+          ]
+        }
+      }
+    }
+  ]
+}
+```
+
+## Supported data types
+
+The following column data types are currently supported:
```

Comment:
Maybe add some details here. I think the concept of "data types" in general is not defined (it depends on the data format), and as the datasets server uses Python, in fact, we are supporting a list of Python (and `datasets`-specific) types.

So: should we say that, with a list of the exact supported Python types?

Or on the contrary only give an overview like:

> strings, float and integers numbers, and the special `ClassLabel` type defined by the `datasets` library

Reply:
i like it the second way! don't want to go into details of underlying types and stuff, rewritten in [46ab3a4](https://github.com/huggingface/datasets-server/pull/1936/commits/46ab3a4d6195b6ec922084c787245ef7c14ccf52)

Reply:
yes, perfectly clear now